Makefile

```
@@ -17,6 +17,7 @@ GIT_COMMIT_HASH ?= $(shell git rev-parse HEAD)
 GIT_TREESTATE=$(shell if [ -n "$(git status --porcelain)" ]; then echo "dirty"; else echo "clean"; fi)
 BUILD_DATE = $(shell date -u +'%Y-%m-%dT%H:%M:%SZ')
 ROOT_DIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+UT_TYPE = --docker
```

```suggestion
UT_ENV= --docker
```

maybe better, means different UT running environment

---

How about  `make ut RUN_WITH_CONTAINER=0` or `make ut RUN_WITH_CONTAINER=1`